src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35341/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:35341/project/issues?id=sample&pullRequest=6326&issues=AWkL66iG7jQHqTntc_8e&open=AWkL66iG7jQHqTntc_8e)

[ISSUE_KEY]: AWkL66iG7jQHqTntc_8e